tools/vm-migrator/subnet.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env python
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This file deals with operations on subnets
+"""
+import googleapiclient.discovery
```

Comment:
Please reorder imports following the order given at https://www.python.org/dev/peps/pep-0008/#imports